traffic_portal/app/src/common/modules/form/deliveryService/form.deliveryService.Steering.tpl.html

```diff
@@ -155,9 +155,7 @@ <h3 ng-if="!open()">Previous Value</h3>
                         </div>
                         </label>
                         <div class="col-md-10 col-sm-10 col-xs-12">
-                            <select id="tenantId" name="tenantId" class="form-control" ng-model="deliveryService.tenantId" ng-options="tenant.id as tenantLabel(tenant) for tenant in tenants" required>
-                                <option disabled hidden selected value="">Select...</option>
-                            </select>
+                            <tree-select handle="tenantId" on-update="deliveryService.tenantId=value" tree-data="[tenants[0]]" initial-value="deliveryService.tenantId"></tree-select>
```

Comment:
Because this no longer uses `ngModel`, it doesn't cause the form to be marked dirty; this means that one can no longer change the tenant of a Delivery Service. At least, not without making any other changes to it so that the form is dirty.

I believe that applies to everywhere this is being used - I only checked the DS and user pages, but that's sufficient to convince me it's systemic.

Reply:
That is what `on-update` does, it was a way to trigger form validation. I'll double check when I get a chance but that _should_ work.

Reply:
You were correct, it should work now.